src/components/Stack/Stack.tsx

```diff
@@ -28,6 +28,8 @@ export interface Props {
   alignment?: Alignment;
   /** Adjust horizontal alignment of elements */
   distribution?: Distribution;
+  /** Fill the available horizontal space in the stack with the item  */
+  fill?: boolean;
```

Comment:
I'm confused about this. Are we adding a prop? I thought this was handled by the distribution prop?